src/main/java/com/github/checkstyle/generatepatchfile/GeneratePatchFile.java

```diff
@@ -157,15 +172,63 @@ public GeneratePatchFile(String repoPath, String testerPath,
             final File reportDir = generate();
             Utils.copyDir(reportDir, new File(destDirName));
 
-            final String patchFileName = String.format("%s-patch-%s-%s.txt", getSimpleRepoName(),
+            final String patchFileName = String.format(PATCH_FILE_FORMAT, getSimpleRepoName(),
                     commitOld.getName().substring(0, commitNameLength),
                     commitNew.getName().substring(0, commitNameLength));
-            if (patchFile.renameTo(new File(destDirName, patchFileName))) {
-                System.out.println("File is moved successful!");
+            if (!(patchFile.renameTo(new File(destDirName, patchFileName)))) {
+                throw new IOException(MOVE_PATCH_FILE_ERROR_MESSAGE);
+            }
+        }
+    }
+
+    private void generateDiffPatchWithGitCommand(String headNum) throws Exception {
```

Comment:
`(String headNum)` - please make it `int` since you expect a number here

Reply:
done